setup.py

```diff
@@ -36,11 +36,12 @@
         "rdkit-pypi==2021.03.5",
         "dask",
         "pandas==1.*",
+        "pdoc3==0.10.*",
```

Comment:
why did you move this requirement to the package requirements ?

Reply:
Because it's necessary if you want to follow the readme, and users may not want to install all the dev requirements.
I could also put it in the `make doc`, what do you think ?

Reply:
I added a "doc" in the extras_require